### PR TITLE
Added provider type to AWS and Host Inventory create cluster pages

### PR DESCRIPTION
**Pull request by o-farag**
Signed-off-by: Omar Farag <ofarag@redhat.com>

Add control plane type to the create cluster catalog as per https://issues.redhat.com/browse/ACM-2357

image

image

### CODE REVIEW

**Comment by o-farag** [comment]
@fxiang1 I noticed that only AWS and Host Inventory have the created catalog cards, while the rest look like this
image

Should I add it to these pages as well?

I also hard coded the types into the title for AWS and Host Inventory because they have their own pages, not sure if that was a good way for me to do it. Also, the types seem to be a bit too large since they're the same size as the title, I was thinking maybe a smaller text or something less aggressive would be better?

**Comment by fxiang1** [comment]
> @fxiang1 I noticed that only AWS and Host Inventory have the created catalog cards, while the rest look like this: image
> 
> Should I add it to these pages as well?
> 
> I also hard coded the types into the title for AWS and Host Inventory because they have their own pages, not sure if that was a good way for me to do it. Also, the types seem to be a bit too large since they're the same size as the title, I was thinking maybe a smaller text or something less aggressive would be better?

@o-farag Right, only AWS and Host inventory has HostedCluster support currently so you only need to change those.
As for the type size, I think it should be fine since we want to make sure the user can clearly see the type and not miss it.

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -740,6 +740,8 @@
     "Control plane pods": "Control plane pods",
     "Control plane status": "Control plane status",
     "Control plane type": "Control plane type",
+    "Control plane type - AWS": "Control plane type - AWS",
```
Comment (fxiang1) [inline frontend/public/locales/en/translation.json:743]: @o-farag For the translations, maybe you can use a variable instead for the type so we can reuse the same translation. Something like this https://github.com/fxiang1/console/blob/main/frontend/src/components/TemplateSummaryModal.tsx#L185

**Review by fxiang1 — CHANGES_REQUESTED** [review]

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (o-farag) [reply frontend/public/locales/en/translation.json]: Alright, changed it to use a variable instead.

**Review by fxiang1 — APPROVED** [review]
LGTM! Thanks Omar!

**Pull request merged**